src/RA_StringUtils.h

```diff
@@ -174,7 +174,7 @@ class StringBuilder
     void Append(const std::string& arg)
     {
         PendingString pPending;
-        pPending.Ref.String = &arg;
+        pPending.Ref = &arg;
```

Comment:
As long as it's in the `std::variant` type list everything is fine, otherwise you'll get intellisense/compiler error.